_episodes/39.md

```diff
@@ -0,0 +1,273 @@
+
+## Demo of the episode: Running Great Expectations on a Trino Data Lakehouse Tutorial
+
+For this episode's demo, you'll need a local Trino coordinator, MinIO instance,
+and Hive metastore and an edge node where various data libraries like Great
+Expectations can run. Clone the 
+[trino-datalake](https://github.com/bitsondatadev/trino-datalake) 
+repository and navigate to the root directory in your cli. Then 
+start up the containers using Docker Compose.
+
+```
+git clone git@github.com:bitsondatadev/trino-datalake.git
+
+cd trino-datalake
+
+docker-compose up -d
+```
+
+The rest of the demo is available in 
+[this markdown tutorial](https://github.com/bitsondatadev/trino-datalake/blob/main/tutorials/expecting-greatness-from-trino.md)
+and is covered in the video demo below.
+
+<div class="youtube-video-container">
+  <iframe width="702" height="405" src="https://www.youtube.com/embed/h6UYOilESfQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+
+## Question of the episode: How can I deploy Trino on Kubernetes without using Helm chart?
+
+[Full question from Trino Slack](https://trinodb.slack.com/archives/C0305TQ05KL/p1660685654979289)
+
+This user was not able to use Helm, due to some restriction in his company. They
+needed the raw kubernetes yaml files to deploy Trino.
+
+_Answer:_ While there are very nice ways that helm offers to directly deploy to
+a service that understands helm charts, you can also use helm on your machine to
+generate all the kubernetes yaml configuration files. This can be done using the
+`helm template` command. See more on this from the 
+[Trinetes episode]({{site.url}}/episodes/31.html) that details this command.
+
+## Events, news, and various links
+
+* [Raft Data Fabric](https://datafabric.goraft.tech/)
+* [Raft Twitter](https://twitter.com/raft_tech)
+* [Raft LinkedIn](https://www.linkedin.com/company/raft-tech/)
+* [Raft Jobs](https://boards.greenhouse.io/raft)
+
+Blogs
+
+* [Trino - SQL to rule them all](https://goraft.tech/2022/08/15/trino-sql-everything.html)
+* [Raft wins USAF SBIR Phase III contract for data centralisation services](https://www.airforce-technology.com/news/raft-wins-usaf-sbir-phase-iii-contract/)
+* [Raft Blog](https://goraft.tech/blog/)
+
+Trino Meetup groups
+ - Virtual
+   - [Virtual Trino Americas](https://www.meetup.com/trino-americas/)
+   - [Virtual Trino EMEA](https://www.meetup.com/trino-emea/)
+   - [Virtual Trino APAC](https://www.meetup.com/trino-apac/)
+ - East Coast (US)
+   - [Trino Boston](https://www.meetup.com/trino-boston/)
+   - [Trino NYC](https://www.meetup.com/trino-nyc/)
+ - West Coast (US)
+   - [Trino San Fransisco](https://www.meetup.com/trino-san-francisco/)
+   - [Trino Los Angeles](https://www.meetup.com/trino-los-angeles/)
+ - Mid West (US)
+   - [Trino Chicago](https://www.meetup.com/trino-chicago/)
```

Comment:
Manfred wanted this simplified. I think you can refer back to what I posted for this on episode 38.